src/app/+display/display.component.ts

```diff
@@ -274,7 +278,14 @@ export class DisplayComponent implements OnDestroy, OnInit {
     return this.showAside(displayView) && displayView.asideLocation === Side.LEFT;
   }
 
-  public showAsideRight(displayView: DisplayView): boolean {
+  public showAsideRight(displayView: DisplayView, individual: Individual): boolean {
+    const excludeTypes: Array<String> = ['Publisher', 'External Organization', 'otherUniversity'];
+
+    const isExcluded: Boolean = individual.type.some(type => excludeTypes.includes(type));
+
+    if (isExcluded)
```

Comment:
For consistency, reducing potential for error, and readability could we avoid the 'single-line block'. Please add curly braces.

Reply:
It might be cleaner to just add `!isExcluded` to the return.

```
return !isExcluded && this.showAside(displayView) && displayView.asideLocation === Side.RIGHT;
```

Reply:
Incorporated suggested return statement, please verify.